composer.json

```
@@ -102,7 +102,7 @@
 		],
 		"build": [
 			"composer require sqlftw/sqlftw --ignore-platform-req=php+",
-			"composer require doctrine/dbal --ignore-platform-req=php+"
+			"composer require doctrine/dbal:^3 --ignore-platform-req=php+"
```

use notation which does not require windows-specific escaping

```suggestion
			"composer require doctrine/dbal:3.* --ignore-platform-req=php+"
```